src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -374,8 +374,8 @@ private static boolean shouldBeAloneOnLine(RightCurlyOption bracePolicy,
                                                String targetSrcLine) {
         return bracePolicy == RightCurlyOption.ALONE
                     && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
-                || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
-                    || details.shouldCheckLastRcurly)
+                    || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
+                        || details.shouldCheckLastRcurly)
```

Comment:
Note the parenthesis, please revert this.

Reply:
What's wrong with this??
```
 && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
 (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
  || details.shouldCheckLastRcurly)

Reply:
![image](https://user-images.githubusercontent.com/812984/158023567-a63dc392-b6e6-48da-b129-0fd05c551227.png)

Reply:
please revert this change.
&& and || are on different priority level of execution. so old formatting was good to show priority of execution.
please revert.

Reply:
ok got it, thank you

done